lib/include/openturns/OTtestcode.hxx

```diff
@@ -379,6 +380,17 @@ inline void assert_almost_equal(const SymmetricMatrix &a, const SymmetricMatrix
   }
 }
 
+inline void assert_almost_equal(const Distribution &a, const Distribution &b, const Scalar rtol = 1.0e-5, const Scalar atol = 1.0e-8, const String errMsg = "")
+{
+  if (a.getImplementation()->getClassName() != b.getImplementation()->getClassName())
```

Comment:
Wrong. A ComposedDistribution with Normal marginals and Normal copula may be equal to a Normal distribution

Reply:
True, but is it a problem for unit tests?